livedata/src/main/java/io/getstream/chat/android/livedata/extensions/ClientExtensions.kt

```diff
@@ -24,26 +21,9 @@ fun Message.getCid(): String =
         this.cid
     }
 
-fun Message.users(): List<User> {
-    val users = mutableListOf<User>()
-    users.add(this.user)
-    for (reaction in this.latestReactions) {
-        reaction.user?.let { users.add(it) }
-    }
-    return users
-}
+fun Message.users() = latestReactions.mapNotNull(Reaction::user) + user
```

Comment:
Those extensions functions should be `internal`

Reply:
did it